components/Forms/NewCollection.js

```diff
@@ -0,0 +1,47 @@
+import updateUserinDb from "@/helpers/updateUserInDb";
+import styled from "styled-components";
+
+import Plus from "/public/icons/svg/plus.svg";
+import StyledInput from "@/components/Styled/StyledInput";
+import AddButton from "@/components/Styled/AddButton";
+import ModalComponent from "../Modal";
+import StyledH2 from "../Styled/StyledH2";
+
+export default function NewCollection({ user, mutateUser, setModal }) {
+  function addNewCollection(event) {
+    event.preventDefault();
+    const collectionName = event.target.collectionName.value;
+    const newCollection = { collectionName: collectionName, recipes: [] };
```

Comment:
you could write this a little shorter:

```suggestion
    const newCollection = { collectionName, recipes: [] };
```